lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -365,8 +365,8 @@ PullImportPaths pullImport(
         MStatus status = UsdMayaUtil::GetMObjectByName(kPullSetName, pullSetObj);
         if (status != MStatus::kSuccess) {
             MString createSetCmd;
-            createSetCmd.format("sets -em -name \"^1s\";", kPullSetName.asChar());
-            MDGModifier& dgMod = MDGModifierUndoItem::create("Pull import pull set creation");
+            createSetCmd.format("sets -em -name \"^1s\";lockNode \"^1s\";", kPullSetName.asChar());
```

Comment:
It depends if the command itself is undoable. The docs for MDGModifier says:

> The command should be fully undoable, otherwise unpredictable behaviour
	may result. For example, assume that the command is a MEL procedure
	which contains some undoable commands and some non-undoable commands.
	When doIt() is first called, the procedure will be executed and all of
	its commands will take effect. If undoIt() is subsequently called, only
	the undoable portions of the script will be undone, which may leave Maya
	in an unstable or inconsistent state. Any subsequent calls to doIt() will
	only redo the commands which were undone, the non-undoable portions
	will not be re-executed.

Reply:
It would be hard to test it anyway, since the undo will delete the set, so you cannot check if it unlocked properly, but then it does not matter since the set is gone, so it should be fine. If the units tests about undo/redo of editAsMaya pass, then it should be fine.